modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -203,9 +160,6 @@ $ {orch-cli} exec -t __<{prod-id-short}-pod-name>__ -n __<{project-context}-name
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} logs __<{prod-id-short}-pod-name>__ -n __<{project-context}-namespace-name>__
-(...)
-Found a custom cert. Adding it to java trust store based on /usr/lib/jvm/java-1.8.0/jre/lib/security/cacerts
```

Comment:
That's a line from the log. It is not relevant any more?

Reply:
It is still relevant. But we have different types (rather sources) of certificates and different messages for them. To avoid confusion (like an user sees different message than in docs) I decided not to add any, just mention about it. I think it would be easy to find certificates related logs.